content/storage-history/history.textile

```diff
@@ -792,7 +793,4 @@ It is important to note that this is not necessarily the order that messages wer
 
 Ably preserves ordering for a specific publisher on a specific channel but, for example, if two publishers in regions A and B publish _message-one_ and _message-two_ simultaneously, then it is very possible that a subscriber in region A will receive _message-one_ before _message-two_, but that a subscriber in region B will receive _message-two_ before _message-one_.
 
-There are instances where messages will not be in canonical global order:
-
-* Recent messages (less than two minutes old) are retrieved from live ephemeral storage and are still ordered by region. They only appear in the canonical global order if you enable "message persistence":/storage-history/storage, which also prevents duplication and missing messages.
-* You choose to retrieve historical messages only up to the point at which a client attaches to a channel. You would typically use this approach to bring a subscriber up to date as part of connection state recovery.
+There are some instances where messages will not be in canonical global order. If you are using continuous history (that is, either a REST history call with @untilAttach=true@, or rewind on a realtime connection), then recent messages (less than two minutes old) are retrieved from live ephemeral storage and are still ordered by region, in order to correctly mesh with the live stream of messages in that region. Older messages retrieved from persisted history (if that is enabled) will be in canonical global order, and Ably will ensure that the transition between the two happens with no duplicates or missed messages.
```

Comment:
```suggestion
There are some instances where messages will not be in canonical global order when using "continuous history":#continuous-history. Continuous history utilizes either a REST history call with the @untilAttach@ parameter set to @true@, or the "rewind":/channels/options/rewind feature on a realtime connection. Messages that are less than two minutes old are retrieved from live ephemeral storage and are ordered only by region. This is to ensure that they correctly mesh with the live stream of messages currently being published in that region. Messages older than two minutes are retrieved from "persisted history":/storage-history/storage#all-message-persistence (if it is enabled). These messages will be in canonical global order. Ably ensures that the transition between those messages retrieved from ephemeral storage, and those from persisted storage, occurs without duplication or missed messages.
```

Just a suggestion to re-word this one slightly, but let me know if I've misunderstood it.